bindings/sundials4py/test/test_sunstepper.py

```diff
@@ -0,0 +1,203 @@
+# -----------------------------------------------------------------
+# Programmer(s): Cody J. Balos @ LLNL
+# -----------------------------------------------------------------
+# SUNDIALS Copyright Start
+# Copyright (c) 2025-2025, Lawrence Livermore National Security,
+# University of Maryland Baltimore County, and the SUNDIALS contributors.
+# Copyright (c) 2013-2025, Lawrence Livermore National Security
+# and Southern Methodist University.
+# Copyright (c) 2002-2013, Lawrence Livermore National Security.
+# All rights reserved.
+#
+# See the top-level LICENSE and NOTICE files for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+# SUNDIALS Copyright End
+# -----------------------------------------------------------------
+# Unit/smoke tests for SUNStepper module
+# -----------------------------------------------------------------
+
+import pytest
+from fixtures import *
+from sundials4py.core import *
+
+
+def make_stepper(sunctx):
+    # Create am empty stepper
+    status, s = SUNStepper_Create(sunctx)
+    assert status == SUN_SUCCESS
+    return s
+
+
+def test_create_stepper(sunctx):
+    s = make_stepper(sunctx)
+    assert s is not None
+
+
+def test_stepper_evolve(sunctx, nvec):
+    s = make_stepper(sunctx)
+    vret = nvec
+    err, tret = SUNStepper_Evolve(s, 1.0, vret)
+    assert isinstance(err, int)
```

Comment:
I guess some of my previous `SUN_SUCCESS` suggestions should be `SUN_ERR_NOT_IMPLEMENTED`
```suggestion
    assert err == SUN_ERR_NOT_IMPLEMENTED
```